assets/style.css

```diff
@@ -199,177 +224,194 @@ header h1 {
   justify-content: center;
 }
 
+#inputArea div div {
+  direction: flex;
+  flex: wrap;
+  flex-direction: row;
+  max-width: 30rem;
+}
+
+/* wordCard (delete comment before final build) */
 #inputArea div div article {
-  background-color: #07E4C9;
-  border-radius: 6px;
-  width: 24;
+  background: linear-gradient(#8a8a8a, #e2e2e2);
+  width: 24rem;
+  border: solid 1px lightgrey;
+  border-radius: 16px;
 }
 
-#submitHomeBtns
-{
+#submitHomeBtns {
   display: flex;
+  flex: wrap;
   flex-direction: row;
-  justify-content: space-between;
+  justify-content: space-evenly;
 }
 
+#wordSubmit {
+  color: black;
+  font-size: 1.25rem;
+  font-weight: 500;
+  background: linear-gradient(#a9bcc4, #d1e6ee);
+  margin: 6px 4px 12px 0;
+  padding: 4px;
+  border-radius: 16px;
+  text-align: center;
+  width: 8rem;
+}
 
-#retryCard {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  margin-top: 4.5rem;
+#startOver {
+  text-align: center;
+  font-size: 1.25rem;
+  font-weight: 500;
+  color: black;
+  background: linear-gradient(#838383, #cacaca);
+  margin: 6px 0 12px 0;
+  padding: 4px;
+  border-radius: 16px;
+  width: 8rem;
+}
+
+#wordText {
+  background-image: none;
+  font-size: 1.2rem;
+  margin-top: 0.2rem;
+  margin-bottom: 0.6rem;
+}
 
+#retryCard {
+  max-width: 30%;
+  min-height: fit-content;
+  max-height: 12rem;
 }
 
+
+/* The Modal (background) */
+.modal {
```

Comment:
overriding bulma styles again

Reply:
Again, you need to include HTML crew in this conversation for this one. I have no way of targeting otherwise. Or you could let me talk to them.

Reply:
That's not even my code....